traffic_ops/testing/api/v14/tenants_test.go

```diff
@@ -62,11 +53,6 @@ func GetTestTenants(t *testing.T) {
 	}
 
 	for _, ten := range testData.Tenants {
-		resp, _, err := TOSession.TenantByName(ten.Name)
```

Comment:
I'm currently getting this error when I run the TO Go API tests:
```
# github.com/apache/trafficcontrol/traffic_ops/testing/api/v14
./tenants_test.go:56:10: resp.Name undefined (type []tc.Tenant has no field or method Name)
./tenants_test.go:57:58: resp.Name undefined (type []tc.Tenant has no field or method Name)
FAIL	github.com/apache/trafficcontrol/traffic_ops/testing/api/v14 [build failed]
```

Was this code meant to be removed?

Reply:
and rebasing to master revealed more api test issues..

Reply:
this is fixed along with a number of other issues in changes I pushed 10/31